pkg/e2e/osd/privileged.go

```diff
@@ -0,0 +1,82 @@
+package osd
+import (
+    "fmt"
+    "log"
+    "time"
+    "github.com/onsi/ginkgo"
+    . "github.com/onsi/gomega"
+    projectv1 "github.com/openshift/api/project/v1"
+    "github.com/openshift/osde2e/pkg/common/config"
+    "github.com/openshift/osde2e/pkg/common/helper"
+    "github.com/openshift/osde2e/pkg/common/state"
+    v1 "k8s.io/api/core/v1"
+    metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+    "k8s.io/client-go/rest"
+)
+func makePod(name string, privileged bool) v1.Pod {
+    return v1.Pod{
+        ObjectMeta: metav1.ObjectMeta{
+            Name: name,
+        },
+        Spec: v1.PodSpec{
+            Containers: []v1.Container{
+                {
+                    Name:  "test",
+                    Image: "registry.access.redhat.com/ubi8/ubi-minimal",
+                    SecurityContext: &v1.SecurityContext{
+                        Privileged: &privileged,
+                    },
+                },
+            },
+        },
+    }
+}
+var _ = ginkgo.Describe("[Suite: service-definition] [OSD] Privileged Containers", func() {
```

Comment:
Just a general thing, but can we add a bit more whitespace in this file separating logically separable parts of the code? Just makes it a little more readable to my eyes.